examples/plot_decoding_tutorial.py

```diff
@@ -167,11 +145,18 @@
 #
 # Let's leave out the 30 last data points during training, and test the
 # prediction on these 30 last points:
-svc.fit(fmri_masked[:-30], conditions[:-30])
+import numpy as np
+n_samples = len(conditions)
+fmri_niimgs_train = index_img(fmri_niimgs, np.arange(0, n_samples - 30))
+fmri_niimgs_test = index_img(fmri_niimgs, np.arange(n_samples - 30, n_samples))
+conditions_train = conditions[:-30]
+conditions_test = conditions[-30:]
 
-prediction = svc.predict(fmri_masked[-30:])
-print((prediction == conditions[-30:]).sum() / float(len(conditions[-30:])))
+decoder.fit(fmri_niimgs_train, conditions_train)
 
+prediction = decoder.predict(fmri_niimgs_test)
+print("Prediction Accuracy: {:.3f}".format(
```

Comment:
Maybe just a short sentence to explain that we print the accuracy as a percentage of test samples predicted successfully

Reply:
@tbng